oauth2client/tools.py

```diff
@@ -201,21 +220,9 @@ def run_flow(flow, storage, flags=None, http=None):
     if not flags.noauth_local_webserver:
         import webbrowser
         webbrowser.open(authorize_url, new=1, autoraise=True)
-        print('Your browser has been opened to visit:')
-        print()
-        print('    ' + authorize_url)
-        print()
-        print('If your browser is on a different machine then '
-              'exit and re-run this')
-        print('application with the command-line parameter ')
-        print()
-        print('  --noauth_local_webserver')
-        print()
+        print(_BROWSER_OPENED_MESSAGE % authorize_url)
```

Comment:
I originally used `.format`, but switched it to `%` to be consistent with the other message in the file.  I can just switch them all here, but `%` is used all over the rest of the code base (~100 lines).  

How do you feel about a PR to convert `%` to `.format()` globally?